docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,31 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 1 March 2023
+
+Frontend version: 1.0.0-20230223.0826 | Backend version: 1.50.0-230224.0351
```

Comment:
our backend version currently deployed to DEV is 1.50.2-230301.0334

Reply:
BE ver changed